test-netconnection: add page (#12654)
tldr-pages/tldr: pages/windows/test-netconnection.md

# Test-NetConnection

> > Displays diagnostic information for a connection.

Reviewer: [Please use the imperative in descriptions](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).
```suggestion
> Display diagnostic information for a connection.
```

> This command can only be used through PowerShell.
> More information: <https://learn.microsoft.com/powershell/module/nettcpip/test-netconnection>.

- Test a connection and display detailed results:

`Test-NetConnection -InformationLevel Detailed`

> - Test a connection to a remote host with port number:

Reviewer: ```suggestion
- Test a connection to a remote host using the specified port number:
```

> `Test-NetConnection -ComputerName {{IP or Host Name}} -Port {{Port Number}}`

Reviewer: > Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word placeholders.

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming

```suggestion
`Test-NetConnection -ComputerName {{ip_or_hostname}} -Port {{port_number}}`
```
